src/containers/other/NFTView/NFTView.tsx

```diff
@@ -22,6 +22,7 @@ import {
 import useCurrency from 'lib/hooks/useCurrency';
```

Comment:
NFTView appears to have a data leak but I've been unable to identify exactly where that is happening!

Reply:
Looks like `useNftData` isn't checking if it's still mounted before setting state. Will just have to add an `isMounted` ref to the hook, and set it on mount/unset it on dismount. Have a look at `useBidData` in `CancelBid` for an example.